packages/core/src/rules/common/security-defined.ts

```diff
@@ -25,6 +27,14 @@ export const OperationSecurityDefined: Oas3Rule | Oas2Rule = () => {
             });
           }
         }
+
+        if (root.security || eachOperationHasSecurity) {
+          return;
+        } else {
+          report({
+            message: `Every API should have security defined on the root level or for each operation.`,
```

Comment:
@adamaltman would use your advise on the message here. Thanks.

Reply:
Sounds good to me.